### ISSUE TITLE
Question regarding fine-tuning for Infrared and Visible Image Fusion (IVIF) without reference images

### ISSUE THREAD

**Issue by HaoYing09** [Henry-Lee-real/DiTFuse#4]
Hi @Henry-Lee-real , thank you for open-sourcing this project.
I am currently trying to fine-tune the model for Infrared and Visible Image Fusion (IVIF). However, I noticed that the model's loss calculation requires a reference image (ground truth). But most IVIF datasets lack reference images.
I was wondering how you handled the fine-tuning process for this task? Did you modify the code , or is there another approach I should follow?
Thanks for your help!

**Comment by Henry-Lee-real** [comment]
This is the key point of our paper (DiT training definitely requires ground truth), so we constructed the M3 training dataset. In this way, we can simulate fusion and also have ground truth available. In addition, to give the model better pixel-level perceptual capability, we also used part of the mean-fusion data as ground truth. The above is mentioned in the Method section.

**Comment by HaoYing09** [comment]
Thank you for your reply. I will review the paper and continue trying to fine-tune the model. Thanks!
